arch/risc-v/src/esp32c3/hardware/esp32c3_rtccntl.h

```diff
@@ -49,13 +49,13 @@
  * write-enable the wdt registers
  */
 
-#define RTC_CNTL_WDT_WKEY_VALUE 0x50D83AA1
+#define RTC_CNTL_WDT_WKEY_VALUE     0x50D83AA1
 
 /* The value that needs to be written to RTC_CNTL_SWD_WPROTECT_REG
  * to write-enable the wdt registers
  */
 
-#define RTC_CNTL_SWD_WKEY_VALUE 0x8F1D312A
+#define RTC_CNTL_SWD_WKEY_VALUE     0x8F1D312A
```

Comment:
```suggestion
#define RTC_CNTL_SWD_WKEY_VALUE     0x8f1d312a
```
It is common practice to use lowercase letters for hex literals.

Reply:
Good catch! Solved.